app/controllers/assignments_controller.rb

```diff
@@ -102,7 +102,7 @@ def find_or_create_wiki
     home_wiki = @course.home_wiki
     language = params[:language].present? ? params[:language] : home_wiki.language
     project = params[:project].present? ? params[:project] : home_wiki.project
-    @wiki = Wiki.find_or_create_by(language: language, project: project) || home_wiki
+    @wiki = Wiki.get(language: language, project: project) || home_wiki
```

Comment:
`find_or_create_by` is built-in method that comes from ActiveModel, while `get` here is an original method in `Wiki`. Although we could rename it `find_or_create_by`, in which case it would override the ActiveModel version.

The key change here is that I updated `get` so that it will handle the special case of wikidata, where it should create a wiki record that has langauge `nil` (even if the user inluded a language).